go/pkg/client/cas.go

```diff
@@ -849,7 +876,11 @@ func (c *Client) readBlobStreamed(ctx context.Context, hash string, sizeBytes, o
 		return n, fmt.Errorf("Failed to finalize writing downloaded data downstream: %v", err)
 	}
 	close(done)
-	if wt.writtenBytes != sz {
+	// We can't reliably calculate hash without the full blob
```

Comment:
nitpick: remove "reliably" from this sentence

Reply:
Done.